test/mochitest/browser_dbg-debug-line.js

```diff
@@ -0,0 +1,38 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+// Test ensures zombie debug lines do not persist
+// https://github.com/firefox-devtools/debugger/issues/7755
+add_task(async function() {
+  const dbg = await initDebugger("doc-debug-line.html");
+
+  // Ensure test files load
```

Comment:
i think this kind of comment could go above `waitForSources` but might be needed next to the call